addons/mail/models/mail_message.py

```diff
@@ -1031,15 +1031,22 @@ def _cleanup_side_records(self):
             'notification_ids': [(5, 0, 0)],
         })
 
+    def _is_empty(self):
```

Comment:
i am not sure why we need to change python code here (not qualified to judge). I guess someone more experimented will have an opinion on this

Reply:
it looks ok, it is just to reuse the code into the 2 methods below without duplicating the complex condition to determine whether it's empty